feg/gateway/services/aaa/base_acct/heartbeat_reset_prego1_15.go

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//go:build !go1.15
+// +build !go1.15
```

Comment:
what is this needed?  You create two binaries?

Reply:
one is "+build go1.15" -> Go 1.15 or newer & one is "+build !go1.15" -> older then Go 1.15